cli/src/commands/jobs.rs

```diff
@@ -175,7 +176,14 @@ pub async fn handle_submission(
         let lockfile = matches
             .value_of("LOCKFILE")
             .ok_or_else(|| anyhow!("Lockfile not found"))?;
-        let res = get_packages_from_lockfile(lockfile)
+
+        // Provide a useful error if the lockfile does not exist.
+        let lockfile_path = Path::new(lockfile);
+        if !lockfile_path.exists() {
+            return Err(anyhow!("Lockfile {:?} does not exist", lockfile));
```

Comment:
Anyhow has a `bail!` macro which replaces `return Err(anyhow!(`, but then it may not be as obvious the function is returning.

Reply:
I can see how that would be useful, but honestly `bail!` sounds to me like it would abort the program.